extract_verbs.sh

```diff
@@ -8,115 +8,140 @@ set -e
-  		;;
-	"estonian" | "et")
-  		LANGUAGE="estonian"
-  		LANGCODE="et"
-  		;;
-	"finnish" | "fi")
-  		LANGUAGE="finnish"
-  		LANGCODE="fi"
-  		;;
-	"french" | "fr")
-  		LANGUAGE="french"
-  		LANGCODE="fr"
-  		;;
-	"galician" | "gl")
-  		LANGUAGE="galician"
-  		LANGCODE="gl"
-  		;;
-	"german" | "de")
-  		LANGUAGE="german"
-  		LANGCODE="de"
-  		;;
-	"italian" | "it")
-  		LANGUAGE="italian"
-  		LANGCODE="it"
-  		;;
-	"latin" | "la")
-  		LANGUAGE="latin"
-  		LANGCODE="la"
-  		;;
-	"mongolian" | "mn")
-  		LANGUAGE="mongolian"
-  		LANGCODE="mn"
-  		;;
-	"polish" | "pl")
-  		LANGUAGE="polish"
-  		LANGCODE="pl"
-  		;;
-	"portuguese" | "pt")
-  		LANGUAGE="portuguese"
-  		LANGCODE="pt"
-  		;;
-	"russian" | "ru")
-  		LANGUAGE="russian"
-  		LANGCODE="ru"
-  		;;
-	"slovak" | "sk")
-  		LANGUAGE="slovak"
-  		LANGCODE="sk"
-  		;;
-	"spanish" | "es")
-  		LANGUAGE="spanish"
-  		LANGCODE="es"
-  		;;
-	"swahili" | "sw")
-  		LANGUAGE="swahili"
-  		LANGCODE="sw"
-  		;;
-	*)
-		echo "Invalid or not supported language for now! QUITTING ..."
-		exit 1
-		;;
+    "bulgarian" | "bg")
+        LANGUAGE="bulgarian"
+        LANGCODE="bg"
+        ;;
+    "chinese" | "zh")
+        LANGUAGE="chinese"
+        LANGCODE="zh"
+        ;;
+    "dutch" | "nl")
+        LANGUAGE="dutch"
+        LANGCODE="nl"
+        ;;
+    "english" | "en")
+        LANGUAGE="english"
+        LANGCODE="en"
+    ;;
+    "estonian" | "et")
+        LANGUAGE="estonian"
+        LANGCODE="et"
+        ;;
+    "finnish" | "fi")
+        LANGUAGE="finnish"
+        LANGCODE="fi"
+        ;;
+    "french" | "fr")
+        LANGUAGE="french"
+        LANGCODE="fr"
+        ;;
+    "galician" | "gl")
+        LANGUAGE="galician"
+        LANGCODE="gl"
+        ;;
+    "german" | "de")
+        LANGUAGE="german"
+        LANGCODE="de"
+        ;;
+    "italian" | "it")
+        LANGUAGE="italian"
+        LANGCODE="it"
+        ;;
+    "latin" | "la")
+        LANGUAGE="latin"
+        LANGCODE="la"
+        ;;
+    "mongolian" | "mn")
+        LANGUAGE="mongolian"
+        LANGCODE="mn"
+        ;;
+    "polish" | "pl")
+        LANGUAGE="polish"
+        LANGCODE="pl"
+        ;;
+    "portuguese" | "pt")
+        LANGUAGE="portuguese"
+        LANGCODE="pt"
+        ;;
+    "russian" | "ru")
+        LANGUAGE="russian"
+        LANGCODE="ru"
+        ;;
+    "slovak" | "sk")
+        LANGUAGE="slovak"
+        LANGCODE="sk"
+        ;;
+    "spanish" | "es")
+        LANGUAGE="spanish"
+        LANGCODE="es"
+        ;;
+    "swahili" | "sw")
+        LANGUAGE="swahili"
+        LANGCODE="sw"
+        ;;
+    *)
+        echo "invalid language parameter: quitting"
+        exit 1
+        ;;
 esac
 
-# Form Wikipedia dump URL
+# download Wikipedia dump
 URL="http://download.wikimedia.org/"$LANGCODE"wiki/latest/"$LANGCODE"wiki-latest-pages-articles.xml.bz2"
-echo "Downloading dump from: $URL"
+echo "[CHECK] downloading Wikipedia dump to " $(pwd)
```

Comment:
to quickly distinguish output Im expecting from my program from the outputs of programs called from within my program. may be removed, as it doesnt do much once you know its working..